eventRate.js

```diff
@@ -63,56 +70,92 @@ function endDraw()
   startTime = undefined;
 }
 
-function addPoint(x, y)
+function addCoalescedPoint(x, y)
 {
   eventCount++;
```

Comment:
this line is in the wrong place now - eventCount is used for the event rate which I think we want to represent the real rate (not the coalesced rate).